src/layout/layout.rs

```diff
@@ -1177,8 +1201,8 @@ mod tests {
             assert_eq!(
                 layout[..],
                 [
-                    Rect::new(0, 0, 1, 0),
                     Rect::new(0, 0, 1, 1),
+                    Rect::new(0, 1, 1, 0),
```

Comment:
This is the only test that changed as a result of changing the weights of `Percentage` and `Ratio`. Either we are implicitly assuming these new weights in our tests or our tests are under specified (likely the latter?).

Reply:
Most Likely under specified. that's the other part of why I'd suggest moving this part of the change to another PR. It allows us to see which parts of the change are related to fixed and file and which parts are changes to the existing code.

Reply:
Removed changes to the weights for `Percentage` and `Ratio`.